osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerPlayer.cs

```diff
@@ -71,10 +70,7 @@ private void load()
 
             ScoreProcessor.ApplyNewJudgementsWhenFailed = true;
 
-            LoadComponentAsync(new GameplayChatDisplay(Room)
-            {
-                Expanded = { BindTarget = LeaderboardExpandedState },
```

Comment:
This being coupled to leaderboard expansion state is a bit of a spoke in the wheel. I simplified this to just use `LocalUserPlaying` which is one half of what was steering the expansion state. Maybe fine?

Reply:
I think it's fine, yeah.

Reply:
Or maybe not?

![osu! 2025-05-07 at 09 36 04](https://github.com/user-attachments/assets/8d48c58c-e5f8-49e4-bb42-0b937137a8dc)

Doesn't stay expanded in `MultiplayerSpectator` anymore.